### PR TITLE
Fix fleet-automation github team name

### PR DESCRIPTION
**Pull request by mellon85**

### CODE REVIEW

File: .ddqa/config.toml [hunk 1]
```diff
@@ -121,7 +121,7 @@ github_labels = ["team/remote-config"]
 jira_project = "FA"
 jira_issue_type = "QA"
 jira_statuses = ["To Do", "In Progress", "Done"]
-github_team = "fleet-automation"
+github_team = "Fleet"
```
Comment (CelianR) [inline .ddqa/config.toml:124]: Are you sure it is with an uppercase `F` ?
```suggestion
github_team = "fleet"
```

Comment (mellon85) [reply .ddqa/config.toml:124]: Is it case insensitive?

Comment (CelianR) [reply .ddqa/config.toml:124]: Sorry I thought it wasn't with the `F` in github. I approve 👍

**Review by CelianR — APPROVED** [review]

**Pull request merged**